Utility functions/plot_bar_SE.m

```diff
@@ -0,0 +1,36 @@
+function [output_OLcontrol,output_OLexp,p] = plot_bar_SE(a)
```

Comment:
Great work here. However, it would be better to use sub-functions to replace the repetitive lines.